domain/model/state/controllerstate.go

```diff
@@ -996,16 +1032,19 @@ AND       u.name = $dbUserName.name
 	name := dbUserName{Name: userName.Name()}
 	modelStmt, err := s.Prepare(q, name, dbModelSummary{})
 	if err != nil {
-		return nil, errors.Annotatef(err, "preparing get model summary for user statement")
+		return nil, errors.Capture(err)
+	}
+
+	type controllerUUID struct {
+		UUID string `db:"uuid"`
 	}
 
-	controllerInfo := dbController{}
+	cu := controllerUUID{}
 	controllerUUIDstmt, err := s.Prepare(`
-SELECT &dbController.*
-FROM controller
-`, controllerInfo)
+SELECT &controllerUUID.* FROM controller
```

Comment:
Doesn't really matter since the table only has 2 columns anyway, but we really just need to select the uuid